portugues/revisa/tenta-consegue/reproduz.md

```diff
@@ -1,4 +1,7 @@
-Voc� envia uma resposta ao desenvolvedor descrevendo os passos para a reprodu��o do bug.
-Voc� aguarda ansiosamente pela resposta, e espera que nova vers�o sem o bug seja lan�ada logo.
+Você envia uma resposta ao desenvolvedor descrevendo os passos para a reprodução do bug.
+Você aguarda ansiosamente pela resposta, e espera que nova versão sem o bug seja lançada logo.
+
+[Após um mês sem obter resposta, você decide contatar novamente o desenvolvedor](sem-resposta/contato-desenvolvedor.md)
+
+[Após um mês sem resposta, você entende que a área de TI é cruel e decide vender coco na praia](ti-cruel/coco-na-praia.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.